packages/lightning-core/common/Input.js

```diff
@@ -10,7 +10,7 @@ export class Input extends React.Component {
   handleBlur = () => this.setState({ focused: false })
   handleClick = () => {
     const { selectOnClick, copyOnClick, onCopy, onClick } = this.props
-    if (copyOnClick) {
+    if (copyOnClick && !this.props.disabled) {
```

Comment:
this is small but maybe wanna stay consistent and declare `disabled` on line 12 with the other props?

`const { selectOnClick, copyOnClick, onCopy, onClick, disabled } = this.props`

`if (copyOnClick && !disabled) {`